recipes/folly/all/conan_deps.cmake

```diff
@@ -0,0 +1,39 @@
+# Set the dependency flags expected by https://github.com/facebook/folly/blob/v2023.12.18.00/CMake/folly-deps.cmake
+
+macro(custom_find_package name var)
```

Comment:
may I suggest `package_name` as a variable name, and `variable_prefix` for `var`? Took me a while to work out what was happening - just a minor comment, though.

Reply:
Renamed parameters (and refactored the implementation to avoid setting variables if we should not to)